docs/src/examples/components/RadioGroup/Types/RadioGroupVerticalExample.shorthand.tsx

```diff
@@ -22,21 +22,37 @@ class RadioGroupVerticalExample extends React.Component {
 
   getItems() {
     return [
-      { name: 'pizza', key: 'Capricciosa', label: 'Capricciosa', value: 'capricciosa' },
+      {
+        name: 'pizza',
+        key: 'Capricciosa',
+        label: 'Capricciosa',
+        value: 'capricciosa',
+      },
       {
         name: 'pizza',
         key: 'Prosciutto',
         label: 'Prosciutto',
         value: 'prosciutto',
         disabled: true,
       },
+      {
+        name: 'pizza',
+        key: 'Pepperoni',
+        label: 'Pepperoni',
+        value: 'pepperoni',
+      },
       {
         name: 'pizza',
         key: 'Custom',
         label: (
           <Text>
             Choose your own{' '}
-            <Input input={{ tabIndex: this.state.inputTabIndex }} inline placeholder="flavour" />
+            <Input
+              input={{ tabIndex: this.state.inputTabIndex }}
+              inline
+              variables={{ inputPadding: '2px 8px', fontSize: '12px' }}
```

Comment:
Is there a better way than hard-coding these values here?

Reply:
Would rather define these with styles - that way they won't be dependent on the theme having to define this variables.